app/metrics.yaml

```diff
@@ -0,0 +1,75 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+nav:
+  browser_menu:
+    type: string
+    description: >
+      String of the item clicked in the menu which includes: Back, Forward, Reload, Exit Firefox
+    send_in_pings:
+      - baseline
+    lifetime: user
+    bugs:
+      - N/A
+    data_reviews:
+      - N/A
+    notification_emails:
+      - katglazko@gmail.com
+    expires: 2100-01-01
```

Comment:
In general, these should be 6mo max out from the current date (and would also need to be listed in the [data review form](https://github.com/mozilla/data-review/blob/master/request.md)).